### DISCUSSION TITLE
Support for multiple monitors

### DISCUSSION THREAD

**Discussion by nicymike** [AlmasB/FXGL#1178] [Q&A]
Hey, I could not find anything about the support for multiple monitors.

Is it possible to
1. get screen number for window (eg. primaryStage) and store it in settings
2. open up the window on the screen number stored in settings (if possible)
without working with screens in JavaFX myself?

**Comment by AlmasB** [comment]
Hi, no there is nothing so far in the codebase.

If you have a JavaFX implementation ready(ish) and would like this to be supported by FXGL, feel free to open an issue.

**Reply by nicymike** [reply]
Thank you. I will try to figure out some code.
